README.md

```diff
@@ -12,14 +12,21 @@ Before issuing a new release, the `Unreleased Changes` section should be renamed
 We use [cargo-release](https://crates.io/crates/cargo-release) to simplify the release process.
 Steps:
 
-1. Start a new branch for the release:
-    * `git checkout -b release-vX.Y.Z`
-    * `git push -u origin release-vX.Y.Z`
-2. Update `CHANGELOG.md` as noted above, and commit your changes.
-3. Run `cargo release --dry-run -vv [major|minor|patch]` and check that the things
+1. Ensure your local `main` branch is up to date - the process might try and push this to update tags.
+   * `git co main`
+   * `git remote update`
```

Comment:
I would have expected the `git pull` to do this automatically, but maybe some folks have it configured not to do that by default?